src/handlers/ComponentHandler.ts

```diff
@@ -56,12 +60,14 @@ export async function ComponentHandler(
 		guild: interaction.guild,
 		guildId: interaction.guildId,
 		user: interaction.user,
+		// @ts-expect-error Idk what this is
 		member: interaction.member,
 		memberPermissions: interaction.memberPermissions,
 		component: component,
 		customId: interaction.customId,
 		arguments: args,
 		values: interaction.isSelectMenu() ? interaction.values : undefined,
+		fields: interaction.isModalSubmit() ? interaction.fields : undefined,
```

Comment:
Undefined is used when variable simply does not exist and null is used when we want to set variable to no value.